### DISCUSSION TITLE
How to get image file size, format and Blobs?

### DISCUSSION THREAD

**Discussion by jeevan-lal** [iseahound/ImagePut#53] [Q&A] [answered]
Thank you for creating and maintaining the ImagePut library! It's been incredibly helpful in my projects, and I appreciate all the hard work and dedication you've put into it. Your efforts have made a significant impact on the AutoHotkey community.

Q1 - How to get image file size, format and Blobs?
Q2-  How to resize/crop image and get resized image  base64 string?

**Comment by iseahound** [comment, marked as answer]
1. You can get the file size using other functions in AutoHotkey
2. `str := ImagePutBase64({image: "something.png", scale: 0.5})`

Read the documentation, this is for actual issues.
